### PR TITLE
Migrate to sbt-typelevel-ci-release

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Comment by armanbilge** [comment]
(This one's also ready for review btw :)

File: .sbtrc [hunk 1]
```diff
@@ -1 +1 @@
-alias boot = ;reload ;project kittens ;iflast shell
+alias boot = ;reload ;project / ;iflast shell
```
Comment (joroKr21) [inline .sbtrc:1]: No idea what this is for

**Review by joroKr21 — APPROVED** [review]
Thanks and sorry for the delay - I'm really busy these days 😅

File: .sbtrc (hunk 1 above)
Comment (armanbilge) [reply .sbtrc:1]: @joroKr21 This `.sbtrc` file? Or the change I made here?

This was a very strange setup and if you don't know what it is I think we should just remove it.

Comment (joroKr21) [reply .sbtrc:1]: Yes the `.sbtrc` file

**Pull request merged**